game_eggs/steamcmd_servers/astro_colony/egg-astro-colony.json

```diff
@@ -0,0 +1,154 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-11-05T17:10:13+01:00",
+    "name": "Astro Colony",
+    "author": "limmek@swedon.se",
+    "description": "Explore endless Universe and create the most efficient constellation of colonies. Build conveyor systems to automate production. Take care of Astronauts and their needs! Establish new colonies as part of intergalactic simulation.",
+    "features": [
+        "steam_disk_space"
+    ],
+    "docker_images": {
+        "wine": "ghcr.io\/parkervcp\/yolks:wine_latest"
+    },
+    "file_denylist": [],
+    "startup": "wine64 .\/AstroColony\/Binaries\/Win64\/AstroColonyServer.exe -log -SteamServerName=\\\"${SERVER_NAME}\\\" -port=${SERVER_PORT} -QueryPort=${SERVER_QUERY_PORT}",
```

Comment:
You missunderstood m; when -port is set and an other port than 7777 is used, you can not connect. remove -port